tests/test_parameter_interpolation.py

```diff
@@ -35,37 +35,36 @@ def test_nonbonded(self):
         lambda_plane_idxs = np.random.randint(low=0, high=2, size=N, dtype=np.int32)
         lambda_offset_idxs = np.random.randint(low=0, high=2, size=N, dtype=np.int32)
 
-        for lamb in [0.0, 0.2, 1.0]:
-            for precision, rtol, atol in [(np.float64, 1e-8, 3e-11), (np.float32, 1e-4, 3e-6)]:
+        sigma_scale = 5.0
 
-                # E = 0 # DEBUG!
-                qlj_src, ref_potential, test_potential = prepare_water_system(
-                    coords, lambda_plane_idxs, lambda_offset_idxs, p_scale=1.0, cutoff=cutoff
-                )
-
-                qlj_dst, _, _ = prepare_water_system(
-                    coords, lambda_plane_idxs, lambda_offset_idxs, p_scale=1.0, cutoff=cutoff
-                )
+        qlj_src, ref_potential, test_potential = prepare_water_system(
+            coords, lambda_plane_idxs, lambda_offset_idxs, p_scale=sigma_scale, cutoff=cutoff
+        )
 
-                qlj = np.concatenate([qlj_src, qlj_dst])
+        qlj_dst, _, _ = prepare_water_system(
+            coords, lambda_plane_idxs, lambda_offset_idxs, p_scale=sigma_scale, cutoff=cutoff
+        )
 
-                print("lambda", lamb, "cutoff", cutoff, "precision", precision, "xshape", coords.shape)
+        qlj = np.concatenate([qlj_src, qlj_dst])
 
-                ref_interpolated_potential = nonbonded.interpolated(ref_potential)
+        ref_interpolated_potential = nonbonded.interpolated(ref_potential)
 
-                test_interpolated_potential = potentials.NonbondedInterpolated(*test_potential.args)
+        test_interpolated_potential = potentials.NonbondedInterpolated(*test_potential.args)
 
-                self.compare_forces(
-                    coords,
-                    qlj,
-                    box,
-                    lamb,
-                    ref_interpolated_potential,
-                    test_interpolated_potential,
-                    rtol=rtol,
-                    atol=atol,
-                    precision=precision,
-                )
+        for precision, rtol, atol in [(np.float64, 1e-8, 3e-11), (np.float32, 1e-4, 3e-6)]:
```

Comment:
This might be for another case, but should we standardize the expected precision for the tests?

Reply:
They're functional form dependent - it's harder to achieve a high level of precision with nonbonded interactions since they're a sum of an extremely large number of largely varying numbers.